### DISCUSSION TITLE
Haven't received my Student Pack and it is more than 5 days

### DISCUSSION THREAD

**Discussion by gautamsah** [community/community#120701] [GitHub Education]
### Hi everyone,

<img width="236" alt="image" src="https://github.com/community/community/assets/75912337/19227f9a-8447-4a0c-b5b7-bf9bfa5b87f9">

**Comment by eddysantiagoo** [comment]
Hi @gautamsah , i had the same issue when i made my verification of github education, i did it by adding my institutional email and a certificate issued by my university indicating that I was studying at that university, but it took me around 1 week to activate the benefits (In some cases I have seen that it takes up to 2 weeks to activate the benefits)

When the benefits are added you'll receive all benefits and email confirmation

If the answer helps you please mark this answer as the correct answer
